SDPi_Supplement/asciidoc/volume1/tf1-ch-b-ref-standards-conformance.adoc

```diff
@@ -33,56 +49,62 @@ ICS Statements per standard may be in a different subfolder and files - along wi
 
-* [[[ref_ws_transfer_2006,WS-Transfer:2006]]] Web Services Transfer (WS-Transfer), W3C Standard, 27 September 2006, available at https://www.w3.org/Submission/WS-Transfer/
+// Appendix B.x
+=== Referenced Standards Conformance
 
-* [[[ref_ihe_pcd_sdpi_white_paper_2019,IHE PCD SDPi White Paper:2019]]] IHE Patient Care Devices (PCD) Service-oriented Device Point-of-Care Interoperability (SDPi) White Paper, Revision 1.1, November 1, 2019.  Available at https://profiles.ihe.net/DEV/[profiles.ihe.net/DEV/].
+// Appendix B.x
+[sdpi_offset=1]
+==== Mapping Foundational Standard Requirements to SDPi Constructs
+#TODO:  Add RI overview & spec / styleguide for how requirements from ICS tables and similar are mapped to where they are satisfied + a link / reference to the general discussion in TF-1 Appendix A and the Referenced Standards model for RI#
 
-* [[[ref_ihe_pcd_sdpi_use_cases_compendium_2019,IHE PCD SDPi Use Cases Compendium:2019]]] IHE Patient Care Devices (PCD) Compendium of Medical Device Oriented Use Cases, Companion to the "Service-oriented Device Point-of-Care Interoperability (SDPi)" White Paper, Revision 1.1, November 1, 2019.  Available at https://profiles.ihe.net/DEV/[profiles.ihe.net/DEV/].
 
-#TODO: DOES ASCIIDOC NOT SUPPORT REPLACEMENT CHARACTERS (SEE NEXT TWO) OR IS IT DONE IN PUBLICATION TO HTML?#
+// Appendix B.x
+==== Overview of Standards Conformity Statements
+#TODO:  Provide a general overview of the topic, including some standards that provide Conformity statements, ICS tables, etc., and those that are moot on the topic - and how all of this is handled in this appendix and TF specification#
 
-* [[[ref_omg_sysml_2_0_spec,OMG SysML ^&#174;^ 2.0]]]  Systems Modeling Language &#8482; (SysML^&#174;^), Version 2.0, Object Management Group (OMG) specification.  Available at https://github.com/Systems-Modeling/SysML-v2-Release/tree/master/doc[github.com/Systems-Modeling/SysML-v2-Release/doc/]
+// ISO/IEEE 11073-10207:2017 ICS
 
-* [[[ref_omg_sysml_2_0_intro_graphical_model,OMG SysML ^&#174;^ Intro-Graphical Notation 2.0]]]  Systems Modeling Language&#8482;  (SysML^&#174;^), Version 2.0, Introduction to the Graphical Model, Object Management Group (OMG) standard.  Available at https://github.com/Systems-Modeling/SysML-v2-Release/tree/master/doc[github.com/Systems-Modeling/SysML-v2-Release/doc/]
+include::conformance-statements/tf1-ch-c-conformance-statement-ieee-11073-10207-2017.adoc[]
 
-* [[[ref_iec_60601_1_8_2020,IEC 60601-1-8:2020]]], IEC 60601-1-8:2006/AMD2:2020, Medical electrical equipment — Part 1-8: General requirements for basic safety and essential performance — Collateral standard: General requirements, tests and guidance for alarm systems in medical electrical equipment and medical electrical systems; Amendment 1:2020, and Amendment 2:2020. available at https://webstore.iec.ch/publication/59648[IEC Webstore] and the https://www.iso.org/standard/41986.html[ISO standards store].
+// Appendix B.x
+[bibliography]
```

Comment:
This block is not a bibliography block, it does not contain any bib refs. Please remove.
```suggestion
```

Reply:
Line removed